### ISSUE TITLE
Cannot set java source and target on JDK 13 (Maven)

### ISSUE THREAD

**Issue by mekortiflette** [stoicflame/enunciate#1045]
Hello

I'm having trouble with the enunciate maven plugin. I used it with my old project with no problem. For my new project, I Updated my JDK to version 13, but the documentation won't generate anymore

I'm trying to generate JAX-RS and Jackson documentation

Here is my pom.xml configuration : 
```xml
<maven.compiler.source>13</maven.compiler.source>
<maven.compiler.target>13</maven.compiler.target>
```
```xml
<dependency>
	<groupId>com.webcohesion.enunciate</groupId>
	<artifactId>enunciate-core-annotations</artifactId>
	<version>${enunciate.version}</version>
</dependency>
```
```xml
<plugins>
	<plugin>
		<groupId>com.webcohesion.enunciate</groupId>
		<artifactId>enunciate-maven-plugin</artifactId>
		<version>${enunciate.version}</version>
		<executions>
			<execution>
				<configuration>
					<source>1.8</source>
					<target>1.8</target>
				</configuration>
				<goals>
					<goal>assemble</goal>
				</goals>
			</execution>
		</executions>
	</plugin>
</plugins>
```

Here is the stacktrace
```[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for application:application-core:war:1.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-surefire-plugin is missing. @ line 362, column 12
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building application Core API 1.0
[INFO] ------------------------------------------------------------------------
[WARNING] The artifact org.mapstruct:mapstruct-jdk8:jar:1.3.1.Final has been relocated to org.mapstruct:mapstruct:jar:1.3.1.Final
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ application-core ---
[INFO] Deleting /home/user/workspaces/JEE/application/target
[INFO] 
[INFO] --- apt-maven-plugin:1.1.3:process (default) @ application-core ---
[INFO] 
[INFO] --- enunciate-maven-plugin:2.13.1:assemble (default) @ application-core ---
[INFO] [ENUNCIATE] Using enunciate configuration at /home/user/workspaces/JEE/application/enunciate.xml
[INFO] [ENUNCIATE] Enabled modules: [c-xml-client, csharp-xml-client, docs, gwt-json-overlay, idl, jackson, java-json-client, java-xml-client, javascript-client, jaxb, jaxrs, jaxws, obj-c-xml-client, php-json-client, php-xml-client, ruby-json-client, spring-web, swagger]
[INFO] [ENUNCIATE] No JAXB XML data types: C XML client will not be generated.
[INFO] [ENUNCIATE] No JAXB XML data types: C# XML client will not be generated.
error: Source option 6 is no longer supported. Use 7 or later.
error: Target option 6 is no longer supported. Use 7 or later.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 6.211 s
[INFO] Finished at: 2020-06-18T16:20:54+02:00
[INFO] Final Memory: 40M/144M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.webcohesion.enunciate:enunciate-maven-plugin:2.13.1:assemble (default) on project application-core: Compile failed of Java JSON client-side classes. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.webcohesion.enunciate:enunciate-maven-plugin:2.13.1:assemble (default) on project application-core: Compile failed of Java JSON client-side classes.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException: Compile failed of Java JSON client-side classes.
	at com.webcohesion.enunciate.mojo.ConfigMojo.execute(ConfigMojo.java:441)
	at com.webcohesion.enunciate.mojo.AssembleBaseMojo.execute(AssembleBaseMojo.java:102)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: com.webcohesion.enunciate.EnunciateException: Compile failed of Java JSON client-side classes.
	at com.webcohesion.enunciate.modules.java_json_client.JavaJSONClientModule.compileClientSources(JavaJSONClientModule.java:353)
	at com.webcohesion.enunciate.modules.java_json_client.JavaJSONClientModule.call(JavaJSONClientModule.java:126)
	at com.webcohesion.enunciate.Enunciate.invokeModules(Enunciate.java:920)
	at com.webcohesion.enunciate.EnunciateAnnotationProcessor.process(EnunciateAnnotationProcessor.java:127)
	at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:1023)
	at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:939)
	at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1267)
	at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1381)
	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1263)
	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:935)
	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.lambda$doCall$0(JavacTaskImpl.java:104)
	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.handleExceptions(JavacTaskImpl.java:147)
	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:100)
	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:94)
	at com.webcohesion.enunciate.Enunciate.run(Enunciate.java:693)
	at com.webcohesion.enunciate.mojo.ConfigMojo.execute(ConfigMojo.java:436)
	... 23 more
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

What I understand is that the java source and target version set is the version 6. I tried to change the version of the maven compiler and the enunciate plugin to 9, 8, 13 and nothing works. The only way to make it work is to run the maven build with jdk8.

Thanks in advance for you help

**Comment by stoicflame** [comment]
We should definitely update the source and target versions, so I'll keep this issue open to track that work.

The workaround is for you to tell Enunciate to use specific source and target versions when compiling java client classes. Example `enunciate.xml` config file:

```xml
<enunciate>
  <modules>
    <java-json-client javac-source="7" javac-target="7"/>
    <java-xml-client javac-source="7" javac-target="7"/>
```

**Comment by mekortiflette** [comment]
GREAT !

Thank you ! You just offered me a loooot of time =D
I also take this opportunity to thank you for the great piece of software you're making :)

**Comment by stoicflame** [comment]
Fixed in `2.13.2`.
